modules/unsupported/cql-json/src/main/java/org/geotools/filter/text/cqljson/CQL2Json.java

```diff
@@ -0,0 +1,111 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2022, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.filter.text.cqljson;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import org.geotools.filter.FilterFactoryImpl;
+import org.geotools.filter.text.cql2.CQLException;
+import org.locationtech.jts.io.ParseException;
+import org.opengis.filter.Filter;
+import org.opengis.filter.expression.Expression;
+
+/**
+ * <b>OGC API CQL2-Json</b>. This class presents the operations available to convert filters into
+ * the CQL2-Json language
+ *
+ * <p>
+ */
+public class CQL2Json {
+
+    private CQL2Json() {
+        // do nothing, private constructor
+        // to indicate it is a pure utility class
+    }
+
+    private static ObjectMapper objectMapper = new ObjectMapper();
+    /**
+     * Generates the CQL2-Json predicate associated to the {@link Filter} object.
+     *
+     * @return CQL2-Json predicate
+     */
+    public static JsonNode toCQL2Json(Filter filter) {
+        FilterToCQL2Json toCQL2Json = new FilterToCQL2Json(objectMapper);
+        return (JsonNode) filter.accept(toCQL2Json, objectMapper.createObjectNode());
+    }
+
+    /**
+     * Generates the CQL2-Json predicate as a string associated to the {@link Filter} object.
+     *
+     * @param filter the filter to convert
+     * @return the CQL2-Json predicate as a string
+     */
+    public static String toCQL2JsonString(Filter filter) {
```

Comment:
```suggestion
    public static String toCQL2(Filter filter) {
```

It's clear that it's a JSON target by the class name.